pkg/api/login_test.go

```diff
@@ -157,10 +158,26 @@ func TestLoginViewRedirect(t *testing.T) {
 			status: 302,
 		},
 		{
-			desc:      "grafana relative url with subpath",
+			desc:      "grafana relative url with subpath without leading slash",
 			url:       "/grafana/profile",
 			appURL:    "http://localhost:3000",
 			appSubURL: "grafana",
+			status:    200,
```

Comment:
Think this test case and others are using wrong assumptions since app URL always ends in `/` and app sub URL always starts with `/`.